spring-jdbc/src/main/java/org/springframework/jdbc/datasource/init/ScriptUtils.java

```diff
@@ -340,6 +407,15 @@ private static void appendSeparatorToScriptIfNecessary(StringBuilder scriptBuild
 		}
 	}
 
+	private static boolean startsWithAny(String script, String[] prefixes, int toffset) {
```

Comment:
```suggestion
	private static boolean startsWithAny(String script, String[] prefixes, int offset) {
```

Reply:
I matched `String.startsWith` but I agree it's an odd name.

Reply:
Yeah, I figured out where you came up with that, but... I still didn't like it. 😇